components/bl-draggable-list-component/src/utils/id.js

```diff
@@ -0,0 +1,10 @@
+export function addId(array) {
+  return array.map((item, idx) => ({ ...item, uniqId: idx }));
```

Comment:
unixId it's not an index.
also this function should be named as enrichWithIds | enrichWithUids